Pod/Classes/SEGGoogleAnalyticsIntegration.m

```diff
@@ -126,6 +126,28 @@ - (void)screen:(SEGScreenPayload *)payload
     SEGLog(@"[[[GAI sharedInstance] defaultTracker] set:%@ value:%@];", kGAIScreenName, payload.name);
 
     GAIDictionaryBuilder *hitBuilder = [GAIDictionaryBuilder createScreenView];
+    
+    NSDictionary *campaign = payload.properties[@"campaign"];
+    if ([campaign isKindOfClass:[NSDictionary class]]) {
+        if ([campaign[@"source"] isKindOfClass:[NSString class]]) {
```

Comment:
Add comment/log to throw error
// Campaign source is the only required campaign field. 

Or we could consider falling back to another value in this case. [From GA](https://developers.google.com/analytics/devguides/collection/ios/v3/campaigns) "If previous call did not set a campaign source, use the hostname as a referrer instead."